src/extension/background-script/actions/liquid/getAddressOrPrompt.ts

```diff
@@ -0,0 +1,70 @@
+import utils from "~/common/lib/utils";
+import type { MessageGetLiquidAddress } from "~/types";
+import { PermissionMethodLiquid } from "~/types";
+
+import state from "../../state";
+import { addPermissionFor, hasPermissionFor } from "../nostr/helpers";
```

Comment:
nit: depending on `nostr` can be misleading, perhaps we could move these helpers into a common module?

Reply:
@louisinger thanks, moved in https://github.com/getAlby/lightning-browser-extension/pull/2583